webrtc.html

```diff
@@ -7520,6 +7520,12 @@ <h3>Simulcast functionality</h3>
         implications of this model is that the <code>addTrack</code> method cannot provide simulcast
         functionality since it does not take <code>sendEncodings</code> as an argument, and therefore cannot
         configure an <code><a>RTCRtpTransceiver</a></code> to send simulcast.</p>
+        <p>The answerer cannot set the <a>simulcast envelope</a> directly. Upon calling the 
+        <code>setRemoteDescription</code> method of the <code><a>RTCPeerConnection</a></code> object, 
+        the <a>simulcast envelope</a> should be configured on the <code><a>RTCRtpTransceiver</a></code>
```

Comment:
Can you change the language to say "this is what happens" rather than "should"? There is a difference between "SHOULD" and "MUST" and this can make you wonder if this is a requirement or if its a bit wishy-washy.

Reply:
Ideally, I prefer avoiding passive language, and speak to what the implementer MUST do, instead of speaking to a user/observer. The spec uses both already quite a lot. Just my 2 cents, as I found it particularly hard to understand here.

Reply:
i have made changes to make it clearer.